### PR TITLE
prevent crash of TickerList in case of no ticker

### LINKED ISSUE #15: Login broken for users without assigned ticker
If you login with a user who doesn't have any ticker assigned, the ticker admin behaves weird. For a short moment, you see some rendered page, but it immediately disappears and then only a white page is displayed.

Would be better to get a warning along the lines of "you don't have a ticker assigned, please ask the admins".

### PR DESCRIPTION
**Pull request by kjoscha**
fixes #15

### CODE REVIEW

File: src/components/TickerList.js [hunk 1]
```diff
@@ -108,6 +108,35 @@ export default class TickerList extends React.Component {
 
     }
 
+    tableBody(tickers) {
```
Comment (0x46616c6b) [inline src/components/TickerList.js]:
```suggestion
    renderList(tickers) {
```

Comment (0x46616c6b) [reply src/components/TickerList.js]: Would recommend to rename it.

**Comment by 0x46616c6b** [comment]
Thanks for this contribution!

File: src/components/TickerList.js (hunk 1 above)
Comment (kjoscha) [reply src/components/TickerList.js]: done

**Pull request merged**